analyzer/main.zeek

```diff
@@ -137,6 +139,27 @@ event pe_section_header(f: fa_file, h: PE::SectionHeader) &priority=1
 	f$pe$section_info_table[h$name]$flags = flag_string;
 }
 
+event pe_export_table(f: fa_file, et: PE::ExportTable) {
+    if ( ! pe_log_export_table ) {
+        return;
+    }
+    # If no functions are exported, skip.
+    if ( ! et?$names ) {
+        return;
+    }
```

Comment:
I assume you made this change to address https://github.com/zeek/spicy-pe/pull/6#discussion_r781887865? With this if a user requested logging of the export table I believe they could still see it missing from the log (if there are no names). This seems confusing.

Could you instead just make the filling of `temp_tbl` conditional below?

Reply:
Yes, I agree. I changed it now.